app/javascript/components/enrollment/Enrollment.js

```diff
@@ -86,13 +80,31 @@ class Enrollment extends React.Component {
     }
   };
 
-  submit(_event, groupMember, reenableSubmit) {
+  submit = (_event, groupMember, reenableSubmit) => {
     window.onbeforeunload = null;
     axios.defaults.headers.common['X-CSRF-Token'] = this.props.authenticity_token;
 
     let diffKeys = Object.keys(this.state.enrollmentState.patient).filter(
       k => _.get(this.state.enrollmentState.patient, k) !== _.get(this.props.patient, k) || k === 'id'
     );
+
+    // Manually add preferred contact method for new enrollment if it is not changed
+    if (!this.editMode && !diffKeys.includes('preferred_contact_method')) {
```

Comment:
I guess this is the first and only dropdown in the enrollment wizard that doesn't have blank/nil as the default. I really feel like there should be a better way to handle "defaults" in the enrollment wizard. At the very least, I think we should do this in Contact.js since we already handle other "auto-filling" there. https://github.com/SaraAlert/SaraAlert/blob/master/app/javascript/components/enrollment/steps/Contact.js#L32

Reply:
fixed with issue below